geps/gep-1709/index.md

```diff
@@ -555,7 +508,81 @@ Creating a pull request to add the `ConformanceReport` will start the
 
 [crd]:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
 
-#### Certification Process
+### Reports submission
+
+Implementers can create as many reports as they want, with different API versions,
+API channels, modes, and implementation versions. The reports can be submitted upstream
+by creating a pull request to the Gateway API repository. The reports folder tree
+is the following:
+
+```console
+|-- conformance/reports
+|   |-- v1.0
+|   |   |-- acme-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v2.13-plain-report.yaml
+|   |-- v1.1
+|   |   |-- acme-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v2.14-plain-report.yaml
+|   |   |   |-- standard-v2.14-with-the-lot-report.yaml
+|   |   |   |-- extended-v2.14-with-the-lot-report.yaml
+|   |   |-- umbrella-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v1.8-plain-report.yaml
+```
+
+The main folder `conformance/reports` contains a set of sub-folders, each for a
+Gateway API version. Implementors will create their project folder in all the Gateway
+API version folders for which they have a supported implementation. The implementors
+folder contains the following set of files:
+
+* `README.md`: contains general information about the implementation and a table
+  with some information, as shown below, to give users a quick overview of
+  the implementation details.
```

Comment:
I think a key goal of `README.md` would be to provide information about how the conformance tests were run and how someone could replicate the results.